kubectl-rook-ceph.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eEuo pipefail
+
+usages() {
+  cat <<EOF
+Usage: kubectl rook [commands] [args] [<flags>]
+Provide common management and troubleshooting tools for the Rook Ceph storage provider as a Krew plugin.
+
+commands:
+  ceph
+args:
+  any ceph argument example: status
+flags:
+  -f  --format   format output in json, json-pretty etc.
+
+EOF
+  exit 0
+}
+
+ceph_command() {
+  echo "${*}"
```

Comment:
Why echoing this?

Reply:
> Why echoing this?

the `ceph command` or basically the `command` we are excuting

Reply:
Ok but is it relevant?

Reply:
so user will run
`kubectl rook ceph status` (unitl the tool is added in krew after that `kubectl rook-ceph ceph status`)
output
```
ceph status
 <then ceph status output>
```
so it just looks good when printing but we can remove it.

Reply:
If a user does `kubectl rook-ceph ceph status -f json-pretty`, a parser like `jq` will have trouble with the output from this `echo`. You could echo to stderr for debug logging, but that is probably unnecessary and may be confusing.